src/parsers/powerflowdata_data.jl

```diff
@@ -828,17 +817,6 @@ function read_dcline!(
     return
 end
 
-function read_dcline!(
```

Comment:
Why this these lines went away? This were besides the point of the AC/DC.

Reply:
Yeah, these functions were repeated. If you check the source code you'll see these functions two times